### PR TITLE
Update macros.md: Add documentation about alias

### PR DESCRIPTION
**Pull request by officialcjunior**
**Detailed description**
Added basic documentation for aliases.

**Closing issues**
Related to #173

### CODE REVIEW

File: src/scripting/macros.md [hunk 1]
```diff
@@ -61,3 +61,27 @@ simple scripting situations. To create a macro that takes arguments you simply a
 ```
 As you can see, the arguments are named by index, starting from 0: $0, $1, ...
 
+# Aliases
+
+radare2 also offerrs aliases which might help you save time by quickly executing your most used commands. They are under `$?`
```
Comment (Maijin) [inline src/scripting/macros.md:66]: May want to use some spellcheck for example grammarly see "offerrs".

File: src/scripting/macros.md [hunk 2]
```diff
@@ -61,3 +61,27 @@ simple scripting situations. To create a macro that takes arguments you simply a
...
+
+radare2 also offers aliases which might help you save time by quickly executing your most used commands. They are under `$?`
+
+The general usage of the feature is: `$alias=cmd`
+
+```
+[0x00404800]> $disas=pdf
+```
+
+The above command will create an alias for `pdf`, which is `$disas`. And the running the command given below would print the disassembly of the main function.
```
Comment (wargio) [inline src/scripting/macros.md:74]: bad English here.

Comment (officialcjunior) [reply src/scripting/macros.md:74]: How about:
'The above command will create an alias `disas` for `pdf`. The following command prints the disassembly of the main function.'

Comment (XVilka) [reply src/scripting/macros.md:74]: LGTM.

Comment (wargio) [reply src/scripting/macros.md:74]: good for me.

**Pull request merged**